dali/operators/reader/loader/video/video_test_base.h

```diff
@@ -36,8 +36,8 @@ class VideoTestBase : public ::testing::Test {
 
   const int FrameSize(int i) const { return Height(i) * Width(i) * Channels(); }
 
-  void ComapreFrames(const uint8_t *frame, const uint8_t *gt, int size, int eps = 1);
-  
+  void ComapreFrames(const uint8_t *frame, const uint8_t *gt, int size, int eps = 10);
```

Comment:
Why fixing the lint you bumped up eps from 1 to 10?

Reply:
It was added to this commit by mistake, but generally eps=1 was not enough. I increased it to test it overnight and if it fails, I'll try to get aarch machine.

Reply:
Understood.